lcpp.lua

```diff
@@ -1377,9 +1375,36 @@ end
 -- @param predefines OPTIONAL a table of predefined variables
 -- @usage out, state = lcpp.compileFile("../odbg/plugin.h", {["MAX_PAH"]=260, ["UNICODE"]=true})
 function lcpp.compileFile(filename, predefines, macro_sources, next, _local)
-	if not filename then error("processFile() arg1 has to be a string") end
-	local file = io.open(filename, 'r')
-	if not file then error("file not found: "..filename) end
+	if type(filename) ~= 'string' then error("processFile() arg1 has to be a string") end

```

Comment:
@m-schmoock if you want to implement something for resolving include path of arbiter environment, I think we should not do it. because usually such an attempt fall into endless issues (like I have a problem with gcc X.X on linux a.b.c) and small patches for work around each environment.

if we provide some help for this kind of problem, I think customizable path finder is better approach. but current api cannot have such kind of configuration, so new API like following may be required. 
```
local pp = lcpp.new({
  path_finder = my_awesome_path_finder,
})
pp:compileFile("my_header_in_original_path.h");
```